src/Microsoft.AspNetCore.Server.Testing/Deployers/NanoDeployer/NanoDeployer.cs

```diff
@@ -0,0 +1,251 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Reflection;
+using System.Diagnostics;
+using System.IO;
+using Microsoft.Extensions.FileProviders;
+using Microsoft.Extensions.Logging;
+using System.Text;
+
+namespace Microsoft.AspNetCore.Server.Testing
+{
+    public class NanoDeployer : ApplicationDeployer
```

Comment:
This isn't plugged into the factory. How will it be used?
https://github.com/aspnet/Hosting/blob/dev/src/Microsoft.AspNetCore.Server.Testing/Deployers/ApplicationDeployerFactory.cs#L32